swatch-producer-aws/src/main/java/com/redhat/swatch/aws/processors/BillableUsageProcessor.java

```diff
@@ -88,9 +88,6 @@ public void process(BillableUsage billableUsage) {
     if (billableUsage.getOrgId() != null) {
       MDC.put("org_id", billableUsage.getOrgId());
     }
-    if (billableUsage.getAccountNumber() != null) {
```

Comment:
The property `account_number` should be removed from the generated class BillableUsage in: swatch-core/schemas/billable_usage.yaml.

Moreover, I've seen that swatch-core/schemas/event.yaml still contains the `account_number` property.

Reply:
I have added another commit to cover the billable_usage and events.